### PR TITLE
Add Client model

### PR DESCRIPTION
**Pull request by arturs105**

### CODE REVIEW

File: Toggl.Ultrawave/Models/Client.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace Toggl.Ultrawave
+{
+    public class Client
+    {
+        public int Id { get; set; }
+
+        public int Wid { get; set; }
```
Comment (willsb) [inline Toggl.Ultrawave/Models/Client.cs]: Should be named `WorkspaceId` instead

File: Toggl.Ultrawave/Models/Client.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+        public string Name { get; set; }
+
+        public string At { get; set; }
```
Comment (willsb) [inline Toggl.Ultrawave/Models/Client.cs]: Shouldn't `At` be a `DateTime` instead?

File: Toggl.Ultrawave.Tests/Models/ClientTests.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
+﻿using System.Threading.Tasks;
+using FluentAssertions;
+using Toggl.Ultrawave.Network;
+using Xunit;
+
+namespace Toggl.Ultrawave.Tests.Models
+{
+    public class ClientTests
+    {
+        public class TheClientModel
```
Comment (willsb) [inline Toggl.Ultrawave.Tests/Models/ClientTests.cs]: Should inherit from the class added by #58 instead, after it gets merged

File: Toggl.Ultrawave/Models/Client.cs (hunk 2 above)
Comment (arturs105) [inline Toggl.Ultrawave/Models/Client.cs]: @willsb I saw, that `User` had a `string` property `At`, did the same for Client and Workspace, but I agree, that a `DateTime` would make more sense.

Comment (willsb) [inline Toggl.Ultrawave/Models/Client.cs]: Good point, we should change those for consistency :)

**Pull request merged**